src/UIObject.cpp

```diff
@@ -0,0 +1,20 @@
+#include "UIObject.h"
```

Comment:
With camel case this should be `UiObject`

Reply:
But UI is short for User Interface..

Reply:
Generally in camel case, acronyms are lower case as well. From my experience, anyways.

I'd prefer you change it, but I'll still approve it even if it's not changed, doesn't matter enough.